origin.spec

```diff
@@ -126,18 +123,11 @@ Requires(post):   systemd
 Requires(preun):  systemd
 Requires(postun): systemd
 Obsoletes:      openshift-node < %{package_refector_version}
+Obsoletes:      tuned-profiles-%{name}-node
```

Comment:
Maybe add `Provides: tuned-profiles-%{name}-node` too?I think that's necessary to ensure that if someone tries to install it the package is installed, necessary because there may be tooling which installed that package.

Reply:
I believe adding this provides will fix the tests.

Reply:
Looks like it didn't.